DTS-STN/c19-benefits-node
# correcting to remove the path from the redirect

# Description

Removes path from redirect

## Test Instructions

1. add path to url to be redirected
2. see if path comes after the url to be redirected to
